tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -898,22 +911,59 @@ def _run_conversion(self, meta_graph_def):
       The optimized GraphDef.
     """
     rewriter_config = get_tensorrt_rewriter_config(
-        conversion_params=self._conversion_params)
+        conversion_params=self._conversion_params, is_v2=True)
     grappler_session_config = config_pb2.ConfigProto()
     grappler_session_config.graph_options.rewrite_options.CopyFrom(
         rewriter_config)
     return tf_optimizer.OptimizeGraph(
         grappler_session_config, meta_graph_def, graph_id=b"tf_graph")
 
+  def _for_each_trt_node(self, graph_def, fn):
+    """Helper method to manipulate all TRTEngineOps in a GraphDef."""
+    for node in graph_def.node:
+      if node.op == _TRT_ENGINE_OP_NAME:
+        fn(node)
+    for func in graph_def.library.function:
+      for node in func.node_def:
+        if node.op == _TRT_ENGINE_OP_NAME:
+          fn(node)
+
   # TODO(laigd): provide a utility function to optimize a ConcreteFunction and
   # use it here (b/124792963).
-  def convert(self):
+  def convert(self, num_calibration_runs=None, calibration_input_fn=None):
     """Convert the input SavedModel in 2.0 format.
 
+    Args:
+      num_calibration_runs: number of runs of the graph with
+        calibration_input_fn during calibration.
+      calibration_input_fn: a function that returns input data as a list or
+        tuple, which will be used to execute the converted signature for
+        calibration. All the returned input data should have the same shape.
+        Example:
+        ```
+        def input_fn():
+          return input1, input2, input3
+        ```
+
+    Raises:
+      ValueError: if the input combination is invalid.
+
     Returns:
       The TF-TRT converted Function.
     """
     assert not self._converted
+
+    if (self._need_calibration and
+        (not num_calibration_runs or not calibration_input_fn)):
+      raise ValueError(
+          "Should specify num_calibration_runs and calibration_input_fn"
```

Comment:
Looks like we need to add space. This is what I am getting:
`Should specify num_calibration_runs and calibration_input_fnbecause INT8 calibration is needed`